src/DssSpell.sol

```diff
@@ -1,5 +1,4 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
-
```

Comment:
nit: I would add an empty comment for spacing

Reply:
done

Reply:
done